LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2022, Woven Planet. All rights reserved.
+Copyright (c) 2019-2022, Toyota Research Institute. All rights reserved.
```

Comment:
Shouldn't we use here only `Copyright (c) 2022, Woven Planet. All rights reserved.`?

Reply:
I saw it but I was not sure. @stonier can you help us here?

Reply:
I'll leave it as it and then proceed to remove it in case it is required in a differen PR.